app/src/main/java/com/concordium/wallet/data/backend/ProxyBackend.kt

```diff
@@ -128,4 +128,7 @@ interface ProxyBackend {
         @Path("accountAddress") accountAddress: String,
         @Query("tokenId") tokenId: String
     ): Call<CIS2TokensBalances>
+
+    @GET
+    fun checkIdentityProviderErrors(@Url url: String): Call<String>
```

Comment:
I find this naming confusing. It is used un this code block

```
    fun startIdentityCreation() {
        viewModelScope.launch(Dispatchers.IO) {
            App.appCore.getProxyBackend().checkIdentityProviderErrors(getIdentityProviderUrl())
                .enqueue(object : retrofit2.Callback<String> {
                    override fun onResponse(call: Call<String>, response: Response<String>) {
                        val foundError = response.errorBody()?.string()
                            ?.contains(CreateIdentityError.ID_PUB.errorMessage) == true
                        if (foundError) {
                            _createIdentityError.value = CreateIdentityError.ID_PUB
                        } else {
                            _createIdentityError.value = CreateIdentityError.NONE
                            _createIdentity.value = true
                        }
                    }

                    override fun onFailure(call: Call<String>, t: Throwable) {
                        _createIdentityError.value = CreateIdentityError.UNKNOWN
                        _createIdentity.value = true
                    }
                })
        }
    }
```

so `onFailure` actually means there are no errors. 

Can’t the logic be turned around?